src/Behat/Behat/Output/Node/EventListener/AST/OutlineTableListener.php

```diff
@@ -37,61 +37,28 @@
  */
 final class OutlineTableListener implements EventListener
 {
-    /**
-     * @var OutlineTablePrinter
-     */
-    private $tablePrinter;
-    /**
-     * @var ExampleRowPrinter
-     */
-    private $exampleRowPrinter;
-    /**
-     * @var SetupPrinter
-     */
-    private $stepSetupPrinter;
-    /**
-     * @var SetupPrinter
-     */
-    private $exampleSetupPrinter;
-    /**
-     * @var OutlineNode
-     */
-    private $outline;
-    /**
-     * @var Setup
-     */
-    private $exampleSetup;
-    /**
-     * @var bool
-     */
-    private $headerPrinted = false;
+    private ?OutlineNode $outline;
```

Comment:
I would add `= null` to avoid an uninitialized state for those properties (both phpstan and psalm are bad at dealing with potentially-uninitialized properties)

Reply:
same for lots of properties for which you added property types btw.

Reply:
Good shout, updated